assets/app/index.html

```diff
@@ -14,34 +14,55 @@
     <!-- build:css(.tmp) styles/main.css -->
     <link rel="stylesheet" href="styles/main.css">
     <!-- endbuild -->
+     <style type="text/css">
+
+
+    </style>
   </head>
-  <body>
+<body class="console-os">
     <!-- Add your site or application content here -->
-    <div id="wrap">
-        <div class="header rcue">
-            <div class="navbar utility">
-                <div class="navbar-inner">
-                    <div class="container">
-                        <a class="brand" href="/"><img src="images/logo-origin.svg" alt="OpenShift Origin"></a>
-                    </div>
-                </div>
-            </div>
-        </div>
-        <div class="container">
-          <div ng-view=""></div>
-        </div>
-    </div>
+    <nav class="navbar navbar-default navbar-pf" role="navigation">
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse-1">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+        <a class="navbar-brand" href="/">
+          <img src="images/logo-origin.svg" alt="OpenShift Origin" />
+        </a>
+      </div>
+      <div class="collapse navbar-collapse navbar-collapse-1">
+        <ul class="nav navbar-nav navbar-utility">
+          <li>
+            <a href="http://docs.openshift.org/latest/welcome/index.html">Documentation</a>
+          </li>
+          <!-- Put this back in when we have a user context to show
+          <li class="dropdown">
+            <a href="#">
+              <span class="pficon pficon-user"></span>
+              Jack Johnson <b class="caret"></b>
+            </a>
+            <!-- <ul class="dropdown-menu">
+              <li>
+                <a href="#">Account</a>
+              </li>
+              <li>
+                <a href="#">Settings</a>
+              </li>
+              <li class="divider"></li>
+              <li>
+                <a href="#">Log out</a>
+              </li>
+            </ul>
+          </li> -->
+        </ul>
+      </div>
+
+    </nav>
 
-    <footer>
-        <section id="copyright">
-            <div class="container">
-                <a href="https://www.redhat.com/">
-                    <img src="images/redhat.svg" alt="Red Hat">
-                </a>
-                <div class="pull-right">Copyright &copy; 2014 Red Hat, Inc.</div>
-            </div>
-        </section>
-    </footer>
+    <div class="" ng-view=""></div>
```

Comment:
what? ng-view (which is how angular knows where the contents of the current main controller's template should be injected)? or the empty class that can be removed